OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -23,6 +23,7 @@ public class HealthInfo : ITraitInfo, UsesInit<HealthInit>
 		public readonly bool NotifyAppliedDamage = true;
 
 		[FieldLoader.LoadUsing("LoadShape")]
+		[Desc("Capsule, Circle and Rectangle are supported.")]
```

Comment:
Modders can also add their own implementations, so hardcoding only these three is a bad idea.